classes/webservice/WebserviceSpecificManagementImages.php

```diff
@@ -1126,7 +1126,7 @@ protected function writePostedImageOnDisk($reception_path, $dest_width = null, $
                         $images_types = ImageType::getImagesTypes($this->imageType);
                         foreach ($images_types as $imageType) {
                             if (!ImageManager::resize($tmp_name, $parent_path.$this->wsObject->urlSegment[2].'-'.stripslashes($imageType['name']).'.jpg', $imageType['width'], $imageType['height'])) {
-                                $this->_errors[] = Tools::displayError('An error occurred while copying image:').' '.stripslashes($imageType['name']);
+                                $this->_errors[] = Context::getContext()->getTranslator()->trans('An error occurred while copying this image:', array(stripslashes($imageType['name'])), 'Admin.Notifications.Error');
```

Comment:
Since the meaning is close, we could replace by 'An error occurred while copying this image: %s' maybe?